packages/AdaptiveCards/Actions/SendActionExecuteResponse.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveCards.Templating;
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Components.AdaptiveCards
+{
+    public class SendActionExecuteResponse : Dialog
+    {
+        [JsonConstructor]
+        public SendActionExecuteResponse([CallerFilePath] string sourceFilePath = "", [CallerLineNumber] int sourceLineNumber = 0)
+            : base()
+        {
+            // enable instances of this command as debug break point
+            this.RegisterSourceLocation(sourceFilePath, sourceLineNumber);
+        }
+
+        [JsonProperty("$kind")]
+        public const string Kind = "Microsoft.Bot.Components.SendActionExecuteResponse";
```

Comment:
const before constructor

Reply:
I'll move it but this is where the SDK defines it at in all of their actions.

---
In reply to: [610217498](https://github.com/microsoft/botframework-components/pull/862#discussion_r610217498) [](ancestors = 610217498)